journeyapps-db/test/unit/QuerySpec.ts

```diff
@@ -1,4 +1,4 @@
-import { query } from '@journeyapps/db';
+import { query } from '../../dist';
```

Comment:
This import does not seem right, I think it needs to be from `@journeyapps/db`

Reply:
That produces an error since it is the `@journeyapps/db` package. 
<img width="895" alt="image" src="https://user-images.githubusercontent.com/51082125/158125807-130fe5fb-5d97-48e2-b1d8-24202c3cd4e4.png">
But I added it as it's own dev dependancy and used a pnpm override to link it to the current source. Does that seem right?

Reply:
lerna also warns about "dependancy cycles" if I do that 
<img width="689" alt="image" src="https://user-images.githubusercontent.com/51082125/158126768-fb3228b1-799e-410d-9c3a-16b058b9a183.png">